guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,389 @@
+   layout = "float-right-desktop -background -shadow"
+%}
+
+Like [lightning wallets](/guide/daily-spending-wallet/backup-and-recovery/cloud-backup/), on-chain wallets with robust silent payments functionality require file backups (cloud or offline). This is because such wallets have valuable [user metadata](/guide/daily-spending-wallet/backup-and-recovery/landing-page/#user-metadata) such as:
+- scan and/or spend keys
+- [tweak data](https://github.com/bitcoin/bips/blob/master/bip-0352.mediawiki#user-content-Tweak_Data)
+- [labels](/guide/how-it-works/silent-payments/#labels)
+- contacts
+
+Applications should allow users to manually note backup information in case they misplace the backup file or the recovery application does not support backup files at all. Like regular wallets, silent payment wallets can also be backed up and restored using a recovery phrase. However, this method may result in longer recovery times as well as loss of valuable metadata.
+
+</div>
+
+{% include tip/recommendation.html %}
+
+Applications should display backup information such as recovery phrase or wallet birthday in the UI even if file backups are being performed. This could reuse existing functionality and serve as a partial manual backup.
+
+{% include tip/close.html %}
+
+
+### Recovery
+
+In addition to recovery methods mentioned [here](/guide/daily-spending-wallet/backup-and-recovery/recovery/), applications may enable manual recovery with data unique to silent payments wallets, such as scan/spend key material.
+
+Allowing users to enter the wallet creation date (wallet birthday) during recovery can help reduce recovery time substantially.
+
+Applications should allow both backup & recovery through multiple methods. This is especially important when backup & recovery features are not standardised – during early stages of adoption and if wallets do not support silent payments.
+
+
+{% include image-gallery.html pages = page.images_recovery %}
+
+## Pros and cons
+
+{% include fact/pros.html %}
+
+- Prevents address reuse without repeated user interactions
+- Enables a better user interaction model centred around users instead of unwieldy addresses
+- Reduces the possibility of mistakes while handling on-chain addresses
+- Allows users to optionally infer payment sources through labels
+- Auto-applied labels improve coin-labelling and coin selection
+- Scanning process is more private since sharing on-chain addresses (or xpubs) is not required
+
+{% include fact/close.html %}
+
+{% include fact/cons.html %}
+
+- Static addresses do not match on-chain addresses in transactions, breaking the conceptual model for seasoned users
+- Scanning for payments doesn’t work well with electrum servers (widely used by wallets)
+
+{% include fact/close.html %}
+
+## Resources
+
+- [BIP-352](https://github.com/bitcoin/bips/blob/master/bip-0352.mediawiki)
+- [https://silentpayments.xyz/](https://silentpayments.xyz/)
+
+### Wallets implementing silent payments
```

Comment:
Bitbox recently supported sending to silent payment addresses as well, being the first hardware device to support the feature.